build.md

```diff
@@ -0,0 +1,145 @@
+# Building Volt
+
+###### Note: This repository can be built with the default SDK but will not be outlined in this guide.
+###### Note: x64 platform builds will also not be outlined in this guide.
+
+All build instructions start the same. Clone a local copy of [mini-source-sdk](https://github.com/Joshua-Ashton/mini-source-sdk). (A minified version of [Source SDK](https://github.com/ValveSoftware/source-sdk-2013) with support for VS 2022)
+```bash
+git clone https://github.com/Joshua-Ashton/mini-source-sdk.git
+```
+
+Following your clone of the mini-source-sdk, navigate to the "version" you would like to build (`sdk2013-mp`, `sdk2013-sp`, `asw`). As an example, the multiplayer version will be used from now on.
+```bash
+cd mini-source-sdk/sdk2013-mp/src/
+```
+
+At this time you should now recursively git clone the VPhysics-Jolt repository into your "src" directory.
+```bash
+git clone https://github.com/Joshua-Ashton/VPhysics-Jolt.git vphysics_jolt --recursive
```

Comment:
I would put `--recursive` before the URL to make it clearer

Reply:
You got it